learnEcl/1800-dedup.md

```diff
@@ -5,9 +5,9 @@ slug: dedup
 
 # DEDUP
 
-The DEDUP function removes duplicates from a dataset based on the defined conditions. The result is dataset with unique values for selected fields.
+The DEDUP function removes duplicates from a dataset based on the defined conditions. The result is a dataset with unique values for selected fields.
 
-**Note** To use DEDUP you dataset must be **sorted**.
+**Note** To use DEDUP your dataset must be **sorted**.
```

Comment:
It is worth calling out why the dataset needs to be sorted:  DEDUP compares only adjacent records.